apps/condo/domains/banking/schema/BankAccount.js

```diff
@@ -0,0 +1,195 @@
+            isRequired: true,
+            type: Select,
+            options: Object.keys(COUNTRIES).join(','),
+        },
+
+        routingNumber: {
+            schemaDoc: 'The routing transit number for the bank account.',
+            type: Text,
+            isRequired: true,
+            hooks: {
+                validateInput: ({ existingItem, resolvedData, addFieldValidationError }) => {
+                    const newItem = { ...existingItem, ...resolvedData }
+
+                    const country = get(newItem, 'country')
+                    const routingNumber = get(newItem, 'routingNumber')
+
+                    const { result, errors } = validateRoutingNumber(routingNumber, country)
+
+                    if ( !result ) {
+                        addFieldValidationError(errors[0])
+                    }
+                },
+            },
+        },
+
+        number: {
+            schemaDoc: 'Bank account number',
+            type: Text,
+            isRequired: true,
+            hooks: {
+                validateInput: ({ existingItem, resolvedData, addFieldValidationError }) => {
+                    const newItem = { ...existingItem, ...resolvedData }
+
+                    const country = get(newItem, 'country')
+                    const number = get(newItem, 'number')
+                    const routingNumber = get(newItem, 'routingNumber')
+
+                    const { result, errors } = validateNumber(number, routingNumber, country)
+
+                    if ( !result ) {
+                        addFieldValidationError(errors[0])
+                    }
+                },
+            },
+        },
+
+        currencyCode: CURRENCY_CODE_FIELD,
+
+        approvedAt: {
+            schemaDoc: 'When the bank account received the status of approved',
+            type: DateTimeUtc,
+            isRequired: false,
+            access: {
+                read: true,
+                create: access.canManageIsApprovedField,
+                update: access.canManageIsApprovedField,
+            },
+        },
+
+        approvedBy: {
+            schemaDoc: 'Who set the approved status for the bank account',
+            type: 'Relationship',
+            ref: 'User',
+            isRequired: false,
+            kmigratorOptions: { null: true, on_delete: 'models.SET_NULL' },
+            access: {
+                read: true,
+                create: access.canManageIsApprovedField,
+                update: access.canManageIsApprovedField,
+            },
+        },
+
+        importId: IMPORT_ID_FIELD,
+
+        territoryCode: {
+            schemaDoc: 'Location code (Classifier of Territories of Municipal Units - OKTMO)',
+            type: Text,
+            isRequired: false,
+        },
+
+        bankName: {
+            schemaDoc: 'Bank name',
+            type: Text,
+            isRequired: false,
+        },
+
+        meta: {
+            schemaDoc: 'Structured metadata',
```

Comment:
Is it possible to add a detailed description here? Or specify gql type for this JSON?

Reply:
It this matched with ADR14 ?

Reply:
Yes, it is matched. Since ADR14 is not implemented yet, im not enforcing strict update rules and et-cetera. 

Added a schemaDoc thou